### ISSUE TITLE
Request comment on the IoU code in eval_cityscapes.py

### ISSUE THREAD

**Issue by chrisliu54** [swamiviv/LSD-seg#2]
Please attach **comment** on the code related to IoU.
The following code is sort of hard to understand:
https://github.com/swamiviv/LSD-seg/blob/e373e896be2de7b8c4a0bfa2a7ab0efc924836fd/code/eval_cityscapes.py#L19-L24

BTW, I got bad evaluation results on cityscapes dataset after training on GTA5 dataset.
While the final mean accuracy for validation on GTA5 is 54.8536%, we got the following evaluation results:
![image](https://user-images.githubusercontent.com/12222396/38682706-5960535e-3e9e-11e8-8788-4a93e2846c09.png)

These are quite different from those listed in your paper:
![image](https://user-images.githubusercontent.com/12222396/38682790-8bf83bba-3e9e-11e8-8665-38603f32ee70.png)

**Comment by swamiviv** [comment]
We will add the comment in the next version. 

(1) The histogram way of computing iou is very standard practice in semantic segmentation. I would refer you to classic repositories such as FCN to familiarize yourself with how the evaluation works.

(2) Since the validation accuracy is good, it is very clear that the label mapping is performed incorrectly. I would like to reiterate here: If you directly train the GTA5 data, the labels are mismatched with CityScapes so you will actually be evaluating the wrong classes. So two things need to happen: (a) The GTA5 training has to happen on "mapped" GTA5 data. (b) When evaluating the code, the info. mapping of cityscapes need to be used to convert the corresponding label maps to a common set of classes. This is already included in our eval code. 

I am going to close this issue now since it is directly a consequence of not training on a proper mapped GTA5 data that corresponds to CityScapes. We will make the mapped data/mapping scripts available soon.

**Comment by YifeiAI** [comment]
Hi, where could we find the mapped data/mapping scripts that convert the ground truth label of synthhia/GTA label matched with the cityscape dataset?
